### PR TITLE
Add pipeline support

### PR DESCRIPTION
**Pull request by solid03**

### CODE REVIEW

File: PSMenu/Public/Show-Menu.ps1 [hunk 1]
```diff
@@ -74,117 +74,125 @@ https://github.com/Sebazzz/PSMenu
...
-    Test-HostSupported
-    Test-MenuItemArray -MenuItems $MenuItems
+    begin {
+        $allMenuItems = [System.Collections.Generic.List[System.Object]]::new()
+    }
+    process {
+        $MenuItems.ForEach({ $allMenuItems.Add($_) })
+    }
+    end {
+        Test-HostSupported
```
Comment (Sebazzz) [inline PSMenu/Public/Show-Menu.ps1]: Shouldn’t this be in `Begin { … }`?

File: PSMenu/Public/Show-Menu.ps1 [hunk 2]
```diff
@@ -74,117 +74,125 @@ https://github.com/Sebazzz/PSMenu
...
         [ConsoleColor] $ItemFocusColor = [ConsoleColor]::Green,
         [ScriptBlock] $MenuItemFormatter = { Param($M) Format-MenuItemDefault $M },
         [Array] $InitialSelection = @(),
         [ScriptBlock] $Callback = $null
     )
 
-    Test-HostSupported
-    Test-MenuItemArray -MenuItems $MenuItems
+    begin {
+        $allMenuItems = [System.Collections.Generic.List[System.Object]]::new()
```
Comment (Sebazzz) [inline PSMenu/Public/Show-Menu.ps1]: Please rename to `$AllMenuItems`

**Review by Sebazzz — CHANGES_REQUESTED** [review]
Could you look into the various reformattings that have been done? These make the diff very cluttered and hard to review. Thanks!

File: PSMenu/Public/Show-Menu.ps1 (hunk 1 above)
Comment (solid03) [reply PSMenu/Public/Show-Menu.ps1]: That makes sense!

**Pull request closed without merging**